ska_builder.py

```diff
@@ -203,9 +213,49 @@ def build_list_packages(pkg_names, args, src_dir, build_dir):
         raise ValueError("Packages {} failed".format(",".join(failures)))
 
 
+def overwrite_skare3_version(current_version, new_version, pkg_path):
+    meta_file = pkg_path / 'meta.yaml'
+    with open(meta_file) as fh:
+        t = jinja2.Template(fh.read())
+    text = (t.render(SKA_PKG_VERSION='$SKA_PKG_VERSION',
+                     SKA_TOP_SRC_DIR='$SKA_TOP_SRC_DIR'))
+    if version.parse(yaml.__version__) < version.parse("5.1"):
```

Comment:
this is (was?) needed because meta.yaml files include `{{ SKA_PKG_VERSION }}`. I was trying to deal with parsing the file while leaving these unchanged. What the code does now is it replaces them by something that can be replaced back using string.Template.

The ideal would be to not replace them to begin with, but the parser was stumbling with these symbols. I will see again if there is a better option.

Reply:
This was a case of things working by coincidence, because we always put the ska3-* meta-packages at the top of the requirement list. Committing fix now.

Reply:
I removed this. If we need to, it can always be added.

Reply:
I meant the yaml version check.

Reply:
ah, I guess it is not needed anymore because we are using the newer version of yaml anyway.

Reply:
I removed the check.